pkg/nuget/parse_testcase.go

```diff
@@ -3,96 +3,96 @@ package nuget
 import "github.com/aquasecurity/go-dep-parser/pkg/types"
 
 var (
-    // docker run --rm -i -t mcr.microsoft.com/dotnet/sdk:latest
-    // cd /usr/local/src
-    // dotnet new mvc
-    // dotnet add package Newtonsoft.Json
-    // dotnet add package NuGet.Frameworks
-    // dotnet restore --use-lock-file
-    // cat packages.lock.json | jq -rc '.dependencies[] | keys[] as $k | "{\"\($k)\", \"\(.[$k] | .resolved)\"},"'
-    NuGetSimple = []types.Library{
-        {"Newtonsoft.Json", "12.0.3"},
-        {"NuGet.Frameworks", "5.7.0"},
-    }
+	// docker run --rm -i -t mcr.microsoft.com/dotnet/sdk":"latest
+	// cd /usr/local/src
+	// dotnet new mvc
+	// dotnet add package Newtonsoft.Json
+	// dotnet add package NuGet.Frameworks
+	// dotnet restore --use-lock-file
+	// cat packages.lock.json | jq -rc '.dependencies[] | keys[] as $k | "{\"\($k)\", \"\(.[$k] | .resolved)\", ""},"'
```

Comment:
Looks like your change broke the command. Please make sure your change works in all languages.